kpe/src/model/bert.rs

```diff
@@ -179,9 +191,21 @@ mod tests {
                 .into_arc_tensor(),
         );
         let valid_mask = vec![false; token_size].into();
+        let valid_embeddings = Array2::<f32>::zeros((0, embedding_size));
+
         assert_eq!(embeddings.collect(valid_mask).unwrap(), valid_embeddings);
     }
 
+    #[test]
+    #[ignore = "run slow test with `cargo test --release -- --ignored`"]
```

Comment:
yeah it's not a nice solution. i hope we can remove it when we get a smaller distilled model from pink.

on the other hand the [rust book](https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested) explicitly states this as a use case:
> Sometimes a few specific tests can be very time-consuming to execute, so you might want to exclude them during most runs of cargo test. Rather than listing as arguments all tests you do want to run, you can instead annotate the time-consuming tests using the ignore attribute to exclude them, as shown here: